### ISSUE TITLE
Some footpaths don't render

### ISSUE THREAD

**Issue by joshuatree0587** [OpenRCT2/OpenRCT2#12855]
**OS:** macOS Catalina
**Version:** 10.15.6
**Commit/Build:** 616b8e5fa on HEAD, DEBUG

The first 3 options for footpaths and queues do not render.

**Steps to reproduce:**
1. Download the latest launcher for macOS (currently v0.0.7)
2. Install the "Good Old Games" version of RollerCoaster Tycoon 2 Triple Thill Pack on a Windows computer
3. Copy the installed game to a directory on the Macintosh
4. Configure the launcher with the location of RollerCoaster Tycoon 2
5. Launch the game (at this point you may already see missing footpaths on the title screen)
6. Create a park and open the path tool
7. Observe that the top 3 options for standard footpaths, and the top 3 options for queues, are blank
8. Placing these footpaths has no visual effect, though they will still function for guests

If you have an RCT 1 installation, you can also simply start any scenario with existing footpaths in the park and they will not render.

**Dump file**

**Screenshots / Video:**

<img width="1600" alt="nopaths-menu" src="https://user-images.githubusercontent.com/1166476/92203130-71a94580-ee4e-11ea-804a-342727679b8c.png">
<img width="1600" alt="nopaths-types" src="https://user-images.githubusercontent.com/1166476/92203143-753ccc80-ee4e-11ea-8269-473f665aad2f.png">

**Save game:**

**Comment by duncanspumpkin** [comment]
Sounds like you've not installed rct2 correctly. Verify your files, specifically the .dat files something is wrong with them.

**Comment by duncanspumpkin** [comment]
Also note the version number we expect in the template is the OpenRCT2 version (probably v0.3.0) and the launcher is not developed by this team so is not relevant.

**Comment by AaronVanGeffen** [comment]
Fwiw, a relevant commit id is listed, though, so they do appear to be using the latest develop version.

**Comment by joshuatree0587** [comment]
You were correct that it was only a problem with my data set. I had originally copied the directory over the network. This time I zipped it up and transferred the archive, and now everything's working flawlessly 🤦.
This issue is now resolved. (Sorry for wasting your time!)

Finally, thank you for making this wonderful app; It'll be a true delight to experience this game again and all the memories it carries with it.
